CONTRIBUTING.md

```diff
@@ -317,6 +319,12 @@ Note that for VSCode, to enable "live" linting of TypeScript (and other) file ty
     ]
 ```
 
+`eslint` can fix trivial lint errors. And `eslint` plugin can do that for you when you save a file by adding this line in your setting.
```

Comment:
```suggestion
`eslint`  can automatically fix trivial lint errors when you save a file by adding this line in your setting.
```

small grammar suggestion, but lgtm otherwise